components/openid/org.wso2.carbon.identity.provider/src/main/java/org/wso2/carbon/identity/provider/AttributeCallbackHandler.java

```diff
@@ -296,9 +291,15 @@ protected void populateClaimValues(String userIdentifier, SAMLAttributeCallback
 
         try {
             if (MapUtils.isEmpty(requestedClaimValues)) {
-                mapValues = connector.getUserClaimValues(
-                        MultitenantUtils.getTenantAwareUsername(userId),
-                        claimList.toArray(claimArray), null);
+                try {
+                    connector = IdentityTenantUtil.getRealm(null, userIdentifier).getUserStoreManager();
+                    mapValues = connector.getUserClaimValues(
+                            MultitenantUtils.getTenantAwareUsername(userId),
+                            claimList.toArray(claimArray), null);
+                } catch (Exception e) {
```

Comment:
Fixed in https://github.com/wso2/carbon-identity/pull/685/files#diff-a6ccb84db77ea9c10c88488d7c2c26e1R300